### ISSUE TITLE
Installing keysnail in Thunderbird on Windows corrupts existing keyconfig installation

### ISSUE THREAD

**Issue by WalterGR** [mooz/keysnail#155]
I installed keysnail in Thunderbird 36.0a1 running on Windows 7.

I had previously installed and configured keyconfig.

Installing keysnail caused the commands shown in the keyconfig settings dialog to be renamed, the keys shown to be renamed, and also caused the key bindings that I had added to keyconfig to not be shown.  No keyconfig bindings work any longer.

These changes persisted after uninstalling keysnail.

In short: keysnail corrupted my keyconfig installation.

Uninstalling keyconfig, restarting Thunderbird, installing keyconfig, and restarting Thunderbird again fixed keyconfig, including once again showing the key bindings that I had added.
